.travis.yml

```diff
@@ -3,14 +3,16 @@ install:
   - pip install tox
   - python3 setup.py egg_info
   - pip install -r aiosmtpd.egg-info/requires.txt
-matrix:
+jobs:
   include:
     - python: "3.6"
       env: INTERP=py36
     - python: "3.7"
       env: INTERP=py37
     - python: "3.8"
       env: INTERP=py38
+    - python: "3.9"
+      env: INTERP=py39
```

Comment:
[Python 3.9 has entered general availability for Travis CI.](https://docs.travis-ci.com/user/languages/python/#python-versions)